packages/apps/stock/src/components/organisms/SmallPropertyCard/SmallPropertyCard.tsx

```diff
@@ -53,13 +46,16 @@ const SmallPropertyCard = ({
 
   return (
     <Card style={[styles.card, style]}>
-      <PopUpOneButton
+      <Alert
         visible={popUp}
-        data={unit == null ? `${_value}` : `${_value} ${unit}`}
         title={title}
-        btnTitle={I18n.t('Auth_Close')}
-        onPress={() => setPopUp(!popUp)}
-      />
+        confirmButtonConfig={{
+          width: 50,
+          title: null,
+          onPress: () => setPopUp(!popUp),
+        }}>
```

Comment:
Please use the times icon here to close the alert
Reduce alert width
Increase content text size